test/MinimalDelegation.4337.t.sol

```diff
@@ -96,31 +93,4 @@ contract MinimalDelegation4337Test is DelegationHandler, TokenHandler, ExecuteHa
         uint256 tokenBalanceAfter = tokenA.balanceOf(address(receiver));
         assertEq(tokenBalanceAfter, tokenBalanceBefore + 1e18);
     }
-
-    function test_handleOps_single_eoaSigner_emits_UserOperationRevertReason() public {
```

Comment:
need to add this back in eventually?

Reply:
this was testing an invalid mode, so its not really possible now since we don't revert in executeUserOp with an invalid mode